### ISSUE TITLE
NVDA does not work correctly in the file picker dialog of Skype 8

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#8592]
### Steps to reproduce:
1. Install Skype 8.
2. Open a chat with somebody.
3. Press Control+Shift+F, to invoke the dialog for picking/opening a file for sending it via Skype. Or just activate the button for adding media or files to the chat (right after the chat entry edit field).
4. Navigate via TAB in the file picking dialog and observe the behavior of NVDA, specially in the list view and tree view controls.

### Actual behavior:
NVDA stays silent or plays its error sound, specially in the tree view and list view controls.

### Expected behavior:
NVDA should read all items in the dialog, as it does normally.

### System configuration:
- Operating system: Windows 10 Pro version 1803 (build 17134.191), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- Skype version: 8.28.76.19 Preview, but the issue is reproducible with the stable 8.27.0.85 as well.
- NVDA version: alpha-15781,d58e1cd4, installed.
- Processor: Intel Core i5-4670 at 3.40GHz.
- RAM Memory: 8.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory + 4065MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC887 at Intel Lynx Point PCH - High Definition Audio Controller.

#### Does the issue still occur after restarting your PC?
Yes, it does.

#### Have you tried any other versions of NVDA?
No, I haven't.

**Comment by k-kolev1985** [comment]
I'm attaching a log file with the errors while the file picker dialog is opened.
[NVDA_Log_2018-08-04.log](https://github.com/nvaccess/nvda/files/2259433/NVDA_Log_2018-08-04.log)

**Comment by ehollig** [comment]
Although, I am not able to reproduce this on my system, here are the interesting parts of the log. CC @josephsl 
DEBUGWARNING - Python warning (13:40:54.410):
C:\Program Files (x86)\NVDA\library.zip\keyboardHandler.py:509: DeprecationWarning: Yield() is deprecated
DEBUGWARNING - Python warning (13:41:01.315):
C:\Program Files (x86)\NVDA\library.zip\api.py:274: DeprecationWarning: Yield() is deprecated

Traceback (most recent call last):
  File "core.pyc", line 465, in run
  File "IAccessibleHandler.pyc", line 883, in pumpAll
  File "IAccessibleHandler.pyc", line 671, in processFocusWinEvent
  File "IAccessibleHandler.pyc", line 687, in processFocusNVDAEvent
  File "baseObject.pyc", line 34, in __get__
  File "baseObject.pyc", line 115, in _getPropertyViaCache
  File "NVDAObjects\IAccessible\__init__.pyc", line 647, in _get_shouldAllowIAccessibleFocusEvent
  File "baseObject.pyc", line 34, in __get__
  File "baseObject.pyc", line 115, in _getPropertyViaCache
  File "NVDAObjects\IAccessible\__init__.pyc", line 1568, in _get_parent
  File "baseObject.pyc", line 34, in __get__
  File "baseObject.pyc", line 115, in _getPropertyViaCache
  File "NVDAObjects\window\__init__.pyc", line 272, in _get_parent
  File "NVDAObjects\__init__.pyc", line 132, in __call__
  File "appModules\skype.pyc", line 231, in event_NVDAObject_init
AttributeError: 'Dynamic_Skype8IAccessibleUIAWindowNVDAObject' object has no attribute 'event_objectID'

ERROR - queueHandler.flushQueue (13:41:10.809):
Error in func _fakeFocus from eventQueue
Traceback (most recent call last):
  File "queueHandler.pyc", line 50, in flushQueue
  File "IAccessibleHandler.pyc", line 819, in _fakeFocus
  File "NVDAObjects\__init__.pyc", line 275, in objectWithFocus
  File "NVDAObjects\__init__.pyc", line 132, in __call__
  File "appModules\skype.pyc", line 231, in event_NVDAObject_init
AttributeError: 'Dynamic_Skype8IAccessibleUIAWindowNVDAObject' object has no attribute 'event_objectID'

ERROR - eventHandler.executeEvent (13:41:11.184):
error executing event: gainFocus on <baseObject.Dynamic_Skype8IAccessibleTreeViewItemOutlineItem object at 0x052DDDB0> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 147, in executeEvent
  File "eventHandler.pyc", line 160, in doPreGainFocus
  File "api.pyc", line 107, in setFocusObject
  File "baseObject.pyc", line 34, in __get__
  File "baseObject.pyc", line 115, in _getPropertyViaCache
  File "NVDAObjects\IAccessible\__init__.pyc", line 1582, in _get_container
  File "baseObject.pyc", line 34, in __get__
  File "baseObject.pyc", line 115, in _getPropertyViaCache
  File "NVDAObjects\__init__.pyc", line 496, in _get_container
  File "baseObject.pyc", line 34, in __get__
  File "baseObject.pyc", line 115, in _getPropertyViaCache
  File "NVDAObjects\IAccessible\__init__.pyc", line 1568, in _get_parent
  File "baseObject.pyc", line 34, in __get__
  File "baseObject.pyc", line 115, in _getPropertyViaCache
  File "NVDAObjects\window\__init__.pyc", line 272, in _get_parent
  File "NVDAObjects\__init__.pyc", line 132, in __call__
  File "appModules\skype.pyc", line 231, in event_NVDAObject_init
AttributeError: 'Dynamic_Skype8IAccessibleUIAWindowNVDAObject' object has no attribute 'event_objectID'

**Comment by k-kolev1985** [comment]
The issue is still present with Skype 8. And since it seams that Skype/Microsoft is forcing users to update from version 7 to version 8, I think this issue needs a little bit more attention. It is not critical, but important enough.

**Comment by leonardder** [comment]
I can not reproduce this. What also concerns me is that the log is talking about a Dynamic_Skype8IAccessibleUIAWindowNVDAObject. NVDA itself doesn't know Skype8 objects.

Does this also occur with add-ons disabled? Could it be the case that you have a custom appModule for skype in your profile?

**Comment by k-kolev1985** [comment]
Oops, stupid me... Yes, Doug Lee's NVDA add-on for Skype causes the issue. I've just disabled it and the problem went away. Let's hope Doug can fix this.

**Comment by leonardder** [comment]
Closing as works for me
